src/omero_rois/library.py

```diff
@@ -134,3 +134,37 @@ def masks_from_label_image(
                                       raise_on_no_mask)
         masks.append(mask)
     return masks
+
+
+def masks_from_3d_label_image(
+      planes, rgba=None, c=None, t=None, text=None,
+      raise_on_no_mask=True):
+    """
+    Create mask shapes from a 3d label image (background=0)
+
+    :param list of numpy.array planes: list of 2D label arrays in z order
+    :param rgba int-4-tuple: Optional (red, green, blue, alpha) colour
+    :param c: Optional C-index for the mask
+    :param t: Optional T-index for the mask
+    :param text: Optional text for the mask
+    :param raise_on_no_mask: If True (default) throw an exception if no mask
```

Comment:
For each plane, this implementation find the masks associated with the greatest available range of labels. Unless the labelled arrays are homogeneous, I suspect `raise_on_no_mask=False` would become the most common scenario.

Reply:
True. I just removed `raise_on_no_mask` parameter.